go.mod

```diff
@@ -5,7 +5,6 @@ go 1.19
 replace github.com/openstack-k8s-operators/dataplane-operator/api => ./api
 
 require (
-	dario.cat/mergo v1.0.0
```

Comment:
don't we need this?
@rebtoor

Reply:
Nope, it's correct its removal since `pkg/deployment/nova.go` has been removed.